src/main/java/dev/blaauwendraad/masker/json/KeyContainsMasker.java

```diff
@@ -66,10 +74,14 @@ public byte[] mask(byte[] input) {
          * This also ensures we can safely check for unescaped double quotes (without masking JSON string values).
          */
         MaskingState maskingState = new MaskingState(input, 1);
+        if (AsciiJsonUtil.isArrayStart(maskingState.byteAtCurrentIndexMinusOne())) {
+            maskingState.expandCurrentJsonPath(0, -1);
```

Comment:
I think here having constant like `int TYPE_ARRAY = -1` would be quite nice

Reply:
Refactored it a little bit